web-client/src/Models/Habit.elm

```diff
@@ -291,22 +291,22 @@ extractCreateHabit addHabitInputData =
                         _ ->
                             Nothing
     in
-    case addHabitInputData.kind of
-        GoodHabitKind ->
-            case ( name, description, unitNameSingular, unitNamePlural, frequency ) of
-                ( Just name, Just description, Just unitNameSingular, Just unitNamePlural, Just frequency ) ->
-                    Just <| CreateGoodHabit <| CreateGoodHabitRecord name description goodHabitTime unitNameSingular unitNamePlural frequency
+        case addHabitInputData.kind of
```

Comment:
These difs are the problem with changing indent on the same PR instead of in it's own PR. If there happens to be a bug too it'll be harder to find it cause of the huge dif. Formatting changes on sep commits